### PR TITLE
ALL UPDATE highperf-limitations for s3 ObjectStorage

### PR DESCRIPTION
**Pull request by sebastienlapierre**

### CODE REVIEW

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/translator.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.translations = yaml.load(fd)
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/banner.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.banners = yaml.load(fd)
WARNING: Empty alt attribute for image 3948.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3949.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3936.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3953.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Other images have empty alt attributes
ERROR: Could not process /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/cloud/storage/highperf-limitations/guide.de-de.md
  | UnicodeDecodeError: 'utf-8' codec can't decode byte 0x8b in position 1: invalid start byte
CRITICAL: RuntimeError: Error encountered
```

File: pages/cloud/storage/highperf-limitations/guide.fr-ca.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ Illimité
 
 1 Gbps / connexion
 
+### Nombre maximum de requetes par seconde en écriture sur un bucket
+
+300 (au délà la qualité de service n'est plus garantie)
```
Comment (Pofilo) [inline pages/cloud/storage/highperf-limitations/guide.fr-ca.md]: au délà -> au-delà

File: pages/cloud/storage/highperf-limitations/guide.fr-ca.md [hunk 2]
```diff
@@ -24,6 +24,10 @@ Illimité
 
 1 Gbps / connexion
 
+### Nombre maximum de requetes par seconde en écriture sur un bucket
```
Comment (OVH-Team-Guides) [inline pages/cloud/storage/highperf-limitations/guide.fr-ca.md]: Please fix this typo: requetes -> requêtes

File: pages/cloud/storage/highperf-limitations/guide.fr-ca.md (hunk 1 above)
Comment (sebastienlapierre) [reply pages/cloud/storage/highperf-limitations/guide.fr-ca.md]: done
Thx

File: pages/cloud/storage/highperf-limitations/guide.fr-ca.md (hunk 2 above)
Comment (sebastienlapierre) [reply pages/cloud/storage/highperf-limitations/guide.fr-ca.md]: done
thx

**Comment by OVH-Team-Guides** [comment]
Thanks for your contribution @sebastienlapierre

**Pull request merged**